onnxconverter_common/float16.py

```diff
@@ -157,6 +157,8 @@ def convert_float_to_float16(model, min_positive_val=1e-7, max_finite_val=1e4,
     queue = []
     value_info_list = []
     node_list = []
+    # key = node, value = graph, used to distinguish global with sub-graph
```

Comment:
Is this line still usefull?

Reply:
You mean the comment? Yes, useful to explain ```node_dict = {}```.